setup.py

```diff
@@ -37,7 +37,7 @@ def run_tests(self):
     packages=['blue'],
     tests_require=['tox'],
     cmdclass={'test': Tox},
-    install_requires=['black==21.7b0', 'flake8==3.8.4'],
+    install_requires=['black==21.7b0', 'flake8>=3.8,<5'],
```

Comment:
I generally dislike ceiling pins because they cause compatibility issues when say `flake8` releases a new major version.  Can we get away with just flooring the `flake8` version at `3.8`?  Are any of the patch releases of flake8 incompatible with this change?

Reply:
I think removing '<5' should be fine